scalability_and_performance/enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,88 @@
+:_content-type: ASSEMBLY
+[id="enabling-workload-partitioning"]
+= Workload partitioning
+include::_attributes/common-attributes.adoc[]
+:context: enabling-workload-partitioning
+
+toc::[]
+
+In resource-constrained environments, you can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+
+The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
+With workload partitioning, you annotate the set of cluster management pods and a set of typical add-on Operators for inclusion in the cluster management workload partition.
+These pods operate normally within the minimum size CPU configuration.
+Additional Operators or workloads outside of the set of minimum cluster management pods require additional CPUs to be added to the workload partition.
+
+Workload partitioning isolates user workloads from platform workloads using standard Kubernetes scheduling capabilities.
+
+The following is an overview of the configurations required for workload partitioning:
+
+* The `install-config.yaml` file requires an additional field: `cpuPartitioningMode`.
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+cpuPartitioningMode: AllNodes <1>
+Compute:
+  - architecture: amd64
+    hyperthreading: Enabled
+    name: worker
+    platform: {}
+    replicas: 3
+controlPlane:
+  architecture: amd64  
+  hyperthreading: Enabled
+  name: master
+  platform: {}
+  replicas: 3
+----
+<1> Sets up a cluster for CPU partitioning at install time. The default value is `None`. 
++
+[NOTE]
+====
+Workload partitioning can only be enabled during cluster installation. You cannot disable workload partitioning post-installation.
+====
+
+* In the performance profile, specify the `isolated` and `reserved` CPUs. 
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: openshift-node-workload-partitioning-worker
+spec:
+  cpu:
+    isolated: 0,1 <1>
+    reserved: "2-3" <2>
+  machineConfigPoolSelector:
+    pools.operator.machineconfiguration.openshift.io/worker: ""
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+----
+<1> Sets the isolated CPUs. Ensure all of the Hyper-Threading pairs match.
+<2> Specifies the workloads to pin to the reserved `CPUSet`. When workload partitioning is enabled, platform containers and platform services such as `systemd`, `CRI-O`, and `Kubelet`  are restricted to these CPUs. All CPUs that are not isolated should be reserved.
+
+* The performance profile: 
```

Comment:
I'm wondering if this bullet list can be redistributed in the spirit of making this into a procedure? My thoughts:
1. Can the first bullet about `spec.cpu.reserved` get incorporated in your code callout for that spec?
2. Should the performance profile be conceptually described before the yaml, perhaps in a second sentence?

Reply:
OK.